Formula/ispc.rb

```diff
@@ -23,13 +22,21 @@ class Ispc < Formula
   depends_on "python@3.11" => :build
   depends_on "llvm@15"
 
+  on_linux do
+    depends_on "tbb"
+  end
+
   fails_with gcc: "5"
 
   def llvm
     deps.map(&:to_formula).find { |f| f.name.match? "^llvm" }
   end
 
   def install
+    # FIXME: Set correct RPATH on macOS
+    inreplace "ispcrt/CMakeLists.txt", "set(CMAKE_INSTALL_RPATH $ORIGIN)",
+                                       "set(CMAKE_INSTALL_RPATH #{loader_path})"
```

Comment:
Switched to draft as this probably should be fixed in `ispc` and replaced with patch.

Reply:
Thank you! I added this info into my upstream issue and will see what they say.

Reply:
I couldn't find your issue in https://github.com/ispc/ispc/issues/created_by/p-linnane. Did they reply yet?